en/10_Contributing/00_Issues_and_Bugs.md

```diff
@@ -105,6 +105,7 @@ The current GitHub labels are grouped into five sections:
 | type/ux | Impact on the CMS user interface |
 | type/docs | A docs change |
 | type/userhelp | A userhelp documentation change |
+| type/i18n | Relates to internatinalisation/translations |
```

Comment:
```suggestion
| type/i18n | Relates to localisation/translations |
```
We don't use the term "internationalisation" anywhere

Reply:
Updated